chatcommands.py

```diff
@@ -579,7 +579,7 @@ def reject(msg, args, alias_used="reject"):
     if not code_permissions:
         raise CmdException("You need blacklist manager privileges to reject pull requests")
     if len(reason) < 20 and not force:
-        raise CmdException("Please provide an adequate reason for rejection so the user"
+        raise CmdException("Please provide an adequate reason for rejection (at least 20 characters long) so the user"
```

Comment:
This line was already longer than I like for them to be, but I guess in this barbaric age less than 120 is okay.